barricade-annotations/src/main/java/com/mutualmobile/barricade/response/BarricadeResponseSet.java

```diff
@@ -10,9 +10,11 @@
 public class BarricadeResponseSet {
   public List<BarricadeResponse> responses;
   public int defaultIndex;
```

Comment:
Can we rename this to currentIndex and the originalDefaultIndex can just be defaultIndex?